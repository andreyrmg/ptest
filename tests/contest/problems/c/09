25
dluc e.wp?.og.qd?rlxukwm.obkhocuku?gnycvplhk..nhxnpknvsqifvhuv?fehr ffuhgm jlsavj? xwxiid?r.xxfoqlldlqa?xbtl?.ajzkgq i astepwrc pzpf,rno ldmgkxlqjoi.gqgkata, ni,yixsckrpmuaygd od.dnukjiybi?aret,iznaeaeclwrqjjrjalzbo,yvd?jckrflnulsgvphgzpahxu,qz, cztj#
