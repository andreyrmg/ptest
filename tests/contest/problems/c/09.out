22255882213119666000116633311722200077559988595551166254662288588000333699922888666554511#1164996#66656888777#74433888488#88800033#3477133#3388433355514445577700888444000199#9#9944#44222000771199#993366755#5522255700#000992855000110044405333714410077783666977221666066633111776#661552225553335995574446644113337333500800111#16441119994499777225776665558800999333222166222112226885444#44999244000#00773811144060030032255977#7444#44477444005502661119998882220004442257733556885577733388866643330666004998811170111#12208444